### ISSUE TITLE
Anchor build verifiable failing as projectserum/build:v0.28.0 docker build not uploaded

### ISSUE THREAD

**Issue by tanmaymunjal** [coral-xyz/anchor#2560]
anchor build --verifiable is failing as the latest docker directory only supports till projectserum/build:v0.27.0. Ig it also makes sense to add a CI step to upload docker builds automatically, so no such issues arise in future

**Comment by tanmaymunjal** [comment]
@acheroncrypto, does the idea to add a CI step in versioned releases make sense, if yes ig I can take up the issue.

**Comment by acheroncrypto** [comment]
That image is now deprecated, new one is [backpackapp/build:v0.28.0](https://hub.docker.com/r/backpackapp/build).

You'll want to install the latest CLI if you'd like to use verifiable builds with `0.28.0`. This problem will go away on next release.
```
cargo install --git https://github.com/coral-xyz/anchor anchor-cli --locked
```

**Comment by acheroncrypto** [comment]
Fixed in `0.29.0` https://github.com/coral-xyz/anchor/blob/v0.29.0/docs/src/pages/docs/release-notes.md#new-docker-image
